GPU/Common/FramebufferCommon.cpp

```diff
@@ -566,7 +564,7 @@ void FramebufferManagerCommon::NotifyRenderFramebufferSwitched(VirtualFramebuffe
 	}
 	textureCache_->NotifyFramebuffer(vfb->fb_address, vfb, NOTIFY_FB_UPDATED);
 
-	// ugly...
+	// ugly... is all this needed?
```

Comment:
Hm, DIRTY_PROJTHROUGHMATRIX is definitely needed if the RT size changes.  Not sure about the others immediately.

-[Unknown]